src/app/content/practiceQuestions/components/FinalScreen.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useDispatch } from 'react-redux';
+import styled from 'styled-components/macro';
+import { textRegularStyle } from '../../../components/Typography';
+import { LinkedArchiveTreeSection } from '../../types';
+import { setSelectedSection } from '../actions';
+import NextSectionMessage from './NextSectionMessage';
+
+// tslint:disable-next-line: variable-name
+const StyledFinalScreen = styled.div`
```

Comment:
For screens that have lower height it displays incorrectly:
![image](https://user-images.githubusercontent.com/31478212/101647719-58a9c500-3a39-11eb-99bd-5a1883b978fc.png)

I've had the same problem with the Empty screen and fixed it with this: https://github.com/openstax/rex-web/pull/959/files#diff-026de43244aacd548960351da3f89d97da04840b19ddef5c41ff15cf79056ac3

Reply:
This is still not fully fixed
![image](https://user-images.githubusercontent.com/31478212/101746053-a1aa5980-3acb-11eb-8eae-3b84027ec32f.png)
![image](https://user-images.githubusercontent.com/31478212/101746322-b25acf80-3acb-11eb-8efb-b6f0586041e4.png)